ui/css/itcss/components/send.scss

```diff
@@ -440,16 +440,6 @@
     background-color: var(--color-background-default);
   }
 
-  &__send-arrow-icon {
-    @include H4;
-
-    color: var(--color-secondary-default);
-    transform: rotate(-45deg);
-    position: absolute;
-    top: -2px;
-    left: 0;
-  }
-
```

Comment:
Not used 

![Screen Shot 2022-05-17 at 2 42 47 PM](https://user-images.githubusercontent.com/8112138/168915414-95b61b1e-0282-432a-b413-3f645f2e2074.png)